main.js

```diff
@@ -0,0 +1,32 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+
+renderer.renderPosts(tweeter.getPosts())
+
+const post = function () {
+    tweeter.addPost($('#input').val())
+    renderer.renderPosts(tweeter.getPosts())
+    $('#input').val('')
+}
+
+$('#posts').on('click', '.delete', function () {
+    let $postId = $(this).closest('.post').data().id
+    tweeter.removePost($postId)
+    renderer.renderPosts(tweeter.getPosts())
+})
+
+$('#posts').on('click', '.btnAddComent', function () {
+    let $postId = $(this).closest('.post').data().id
```

Comment:
this variable holds a number in it (the post id) and not a jQuery object, so in this case, you shouldn't use the $ in the beginning on the variable name